Makefile

```diff
@@ -35,4 +35,7 @@ install-clippy:
 	rustup component add clippy --toolchain $(RUSTV)
 
 check-clippy:	install-clippy
-	cargo +$(RUSTV) clippy --all-targets  -- -D warnings
+	cargo +$(RUSTV) clippy --all --all-targets --all-features -- \
+		-D warnings \
+		-A clippy::upper_case_acronyms \
```

Comment:
How many instances of `upper_case_acronyms` were there or is this due to rust 1.51?

Reply:
This is due to 1.51, and I don't know for sure how many instances there are because they do not show up on my local clippy even without the `-A` tag, and when it fails on CI I think it does not show all of them at once, I'm not sure